PayloadPkg/Include/Library/PayloadLib.h

```diff
@@ -20,6 +20,13 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 
 #define  PLD_GDATA_SIGNATURE     SIGNATURE_32('P', 'L', 'D', 'G')
 
+//
+// These MACRO definations should be used only after feature config data is set in stage1b.
+//
+#define ACPI_FEATURE_ENABLED()   (GetFeatureCfg() & FEATURE_ACPI)
+#define ACPI_ENABLED()           (FeaturePcdGet (PcdAcpiEnabled) && ACPI_FEATURE_ENABLED())
+#define MEASURED_BOOT_ENABLED()  (FeaturePcdGet (PcdMeasuredBootEnabled) && (GetFeatureCfg() & FEATURE_MEASURED_BOOT) && ACPI_FEATURE_ENABLED())
+
```

Comment:
To avoid duplication, I think we should move these definitions from BootloaderCoreLib.h to BootloaderCommonLib.h, so they could be used in both sbl core and SBL payload.